Makefile

```diff
@@ -29,6 +29,7 @@ SHELL:=/bin/bash
 PYTHON?=python3
 PYTHON_PIP=$(PYTHON) -m pip
 
+ELYRA_VERSION:=$$(python3 -c "import elyra._version; print(elyra._version.__version__)")
```

Comment:
Should the `python3` here reference `$(PYTHON)` instead in case folks don't have a `python3` instance?

This approach captures the version prior to any targets and pulls the version information from the elyra package that is currently _installed_.  This seems susceptible to potentially capturing an older version.  Might a grep-style approach against the local `_version.py` be more definitive?  E.g., something like...
```bash
grep __version__ elyra/_version.py | awk '{print $3}' | sed s/\"//g
```

Reply:
updated version capture, used a slightly simplified cut with quote delimiter, seems to do the trick